docs/zh-CN/sumo.md

```diff
@@ -57,3 +57,4 @@
 | Rocket | Rocket | 不翻译 |
 | Thunderbird | Thunderbird | 不翻译 |
 | Webmaker | Webmaker | 不翻译 |
+| Transition | 迁移 | 投票决定 |
```

Comment:
This list is brand related. Is `Transition` a brand used somewhere?

Reply:
This list is not all about brand. It is used as a dictionary for the translators.

Recently, we got a request to unify the translation of Transition because it is used for Pocket account articles. We reached the conclusion of the above translation so we keep it recorded here as reference.

Reply:
Thanks for providing some context. Could you get someone with translator/management from the community to review this update, like how it's done in Pontoon? Once confirmed, I will merge the request. Thank you!